sdcm/tester.py

```diff
@@ -345,7 +345,11 @@ def __init__(self, *args):  # pylint: disable=too-many-statements,too-many-local
         if self.params.get("use_ldap"):
             self._init_ldap()
 
-        start_events_device(log_dir=self.logdir, _registry=self.events_processes_registry)
+        # Cover multi-tenant configuration. Prevent event device double initiate
+        if kwargs.get("start_events_device", True):
+            start_events_device(log_dir=self.logdir,
+                                _registry=getattr(self, "_registry", None) or self.events_processes_registry)
```

Comment:
I think that it is not needed to add additional parameter. Would be enough just to check the number of tenants to make the decision.

Reply:
actually it's not needed anymore. removed